docs/user-guide.rst

```diff
@@ -544,6 +544,246 @@ For AWS providers, use :meth:`aws.Credentials.from_info
         ['https://www.googleapis.com/auth/cloud-platform'])
 
 
+External credentials (Workforce identity federation)
+++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+`Workforce identity federation`_ lets you use an external identify provider
+(IdP) to authenticate and authorize a workforce—a group of users, such as
+employees, partners, and contractors—using IAM, so that the users can access
+Google Cloud services. Workforce identity federation extends Google Cloud's
+identity capabilities to support syncless, attribute-based single sign on.
+
+With workforce identity federation, your workforce can access Google Cloud
+resources using an external identity provider (IdP) that supports OpenID
+Connect (OIDC) or SAML 2.0 such as Azure Active Directory (Azure AD), Active
```

Comment:
we can add AWS as well here?

Reply:
AWS is not OIDC IIRC

Reply:
But AWS is supported right? We have AWS specific WIF code

Reply:
Workforce identity federation does not support AWS.